lib/go-tc/service_category.go

```diff
@@ -37,3 +39,25 @@ type ServiceCategory struct {
 	LastUpdated TimeNoMod `json:"lastUpdated" db:"last_updated"`
 	Name        string    `json:"name" db:"name"`
 }
+
+// ServiceCategoriesResponseV50 is a list of Service Categories as a response.
+type ServiceCategoriesResponseV50 struct {
+	Response []ServiceCategoryV50 `json:"response"`
+	Alerts
+}
+
+// ServiceCategoryResponseV50 is a single Service Category response for Update and Create to
+type ServiceCategoryResponseV50 struct {
+	Response ServiceCategoryV50 `json:"response"`
+	Alerts
+}
+
+// ServiceCategoryV50 holds the name and last updated time stamp.
+type ServiceCategoryV50 struct {
+	LastUpdated time.Time `json:"lastUpdated" db:"last_updated"`
+	Name        string    `json:"name" db:"name"`
+}
+
+type ServiceCategoriesResponseV5 = ServiceCategoriesResponseV50
+type ServiceCategoryResponseV5 = ServiceCategoryResponseV50
+type ServiceCategoryV5 = ServiceCategoryV50
```

Comment:
these exported types should have associated GoDoc comments

Reply:
Added GoDoc comments.